components/Navbar.tsx

```diff
@@ -4,7 +4,7 @@ import ToggleSwitch from "./ToggleSwitch";
 export default function LoadingAnimation() {
   return (
     <div className={styles.navbar}>
-      <h1 className={styles.title}>Terralics</h1>
+      <object data="/logo_text_white_box.svg" className={styles.title}></object>
```

Comment:
Heb geen idee wat `<object>` is 😆
Tenzij er een specifieke reden voor was, kunnen we mss best een [Next Image](https://nextjs.org/docs/basic-features/image-optimization) gebruiken

Reply:
Done!